packages/server/src/lib/targeting.ts

```diff
@@ -62,7 +62,12 @@ export const userIsInTest = <V extends Variant>(test: Test<V>, mvtId: number): b
 export const audienceMatches = (
     showSupportMessaging: boolean,
     testAudience: UserCohort,
+    lastOneOffContributionDate?: string,
 ): boolean => {
+    if (lastOneOffContributionDate) {
+        // Recent contributors are excluded from all message tests
+        return !isRecentOneOffContributor(new Date(lastOneOffContributionDate));
+    }
```

Comment:
If they are not a recent contributor, shouldn't we still check the audience?

Reply:
good point! I'm just adding some unit tests and will then fix the logic